README.md

```diff
@@ -802,3 +807,4 @@ means butterfly in Romanian; A creature you might expect to see in Fantasy Land.
 [22]: https://github.com/Avaq/Fluture/wiki/Comparison-to-Promises
 [23]: https://vimeo.com/106008027
 [24]: https://github.com/fantasyland/fantasy-land#bifunctor
+[25]: https://github.com/fantasyland/fantasy-land/tree/1.0.0
```

Comment:
I wish it could be `^1.0.0`, maybe I should just use `master` and pin it once a `2.0` comes out.